modules/customizing-the-web-console-URL.adoc

```diff
@@ -53,6 +53,12 @@ spec:
 
 . Save the file to apply the changes.
 
+[NOTE]
+====
+Add a DNS record for the custom console route pointing to the application ingress load balancer.
```

Comment:
The current wording can be interpreted in 2 ways: 
- "For the custom console route that points to the application ingress load balancer, add a DNS record"
- "For the custom console route, add a DNS record that points to the application ingress load balancer"

Consider revising the sentence for clarity.

Reply:
Hi @lahinson,

This looks good to me:
"For the custom console route, add a DNS record that points to the application ingress load balancer"

Reply:
That is clearer, thanks! I'll mark this peer review as done, assuming you'll make the update to that note.

Reply:
Done